### ISSUE TITLE
Supprimer un tag

### ISSUE THREAD

**Issue by LeoTexier** [dixplo/BigBrainsBoard-Project#46]
**Test scenario**

> Supprimer un tag

**Test steps**

> URL de départ: https://board-leotexier.herokuapp.com/projects
> Cliquer sur l'icone de "bloc note" (où est écrit "Board") sur l'un des projets.
> Cliquer sur bouton avec l'icone "plus" sur la page, sur l'une des stories.
> Cliquer sur l'icone en forme de croix sur la page sur le tag à supprimer.

**Expected results**

> Retour à la gestion des tags sans le tag supprimé.

**Testers**

> @goupilt

**Comment by jcheron** [comment]
oui, mais l'url de départ, on ne va pas y aller directement, en tapant des ids de projet et de story qu'on ne connait pas.
Donnez plutôt la procédure pour y aller.

**Comment by goupilt** [comment]
testé et validé par le testeur
